docs/mathematics/prime-number.mdx

```diff
@@ -0,0 +1,125 @@
+---
+title: Prime Number
+slug: /mathematics/prime-number
+---
+
+## Introduction
+
+Prime numbers are the positive integers having only two factors, 1 and the integer itself.
+
+## Explanation
+
+If `p` is a prime, then it’s only factors are necessarily 1 and p itself. Any number which does not follow this is termed as composite numbers, which means that they can be factored into other positive integers.
```

Comment:
This line seems to be plagiarized from an online source. Please don't copy a single line from other sources. Use your own sentences (It isn't hard)